cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -116,6 +116,9 @@ type ScaleUpFailure struct {
 	Time      time.Time
 }
 
+// BackoffReasonStatus contains information about backoff status and reason
+type BackoffReasonStatus map[string]int
```

Comment:
Since the values here will always be either 0 or 1, maybe `map[string]bool` would make more sense?

Reply:
Modifications done and PR submitted